src/android/toga_android/widgets/_pickers.py

```diff
@@ -0,0 +1,113 @@
+from ..libs.android.view import OnClickListener, View__MeasureSpec
+from ..libs.android.widget import EditText
```

Comment:
Pulling these utility classes into a standalone shared module is a good idea; however, convention inside Toga is to put these in a `widgets/internal` submodule, rather than using a 'protected' module name. See the Cocoa backend for an example of this.

Reply:
Moved to internal